draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -478,9 +485,9 @@ Procedure:
 
 ### Proof Generation (ProofGen)
 
-The ProofGen operation creates BBS proof, which is a zero-knowledge, proof-of-knowledge of a BBS signature, while optionally disclosing any subset of the signed messages. Validating the proof (see ProofVerify defined in (#proof-verification-proofverify)) guarantees authenticity and integrity of the header and disclosed messages, as well as knowledge of a valid BBS signature.
+The ProofGen operation creates a BBS proof, which is a zero-knowledge, proof-of-knowledge of a BBS signature, while optionally disclosing any subset of the signed messages. Validating the proof (see ProofVerify defined in (#proof-verification-proofverify)) guarantees authenticity and integrity of the `header` and disclosed messages, as well as knowledge of a valid BBS signature.
```

Comment:
ProofGen also uses the `presentation_header` and ProofVerify uses that and `header` and disclosed messages...